tenable/io/v3/vm/editor/schema.py

```diff
@@ -0,0 +1,31 @@
+from marshmallow import Schema, fields, validate
+
+
+class EditorSchema(Schema):
+    '''
+    Schema for editor
+    '''
+    etype = fields.Str(
+        required=True,
+        validate=validate.OneOf(['scan', 'policy', 'scan/policy'])
+    )
+
+
+class EditorAuditSchema(Schema):
+    '''
+    Schema for audit file
+    '''
+    etype = fields.Str(
+        required=True,
+        validate=validate.OneOf(['scan', 'policy'])
+    )
+
+
+class EditorTemplateSchema(Schema):
+    '''
+    Schema for editor template
+    '''
+    etype = fields.Str(
+        required=True,
+        validate=validate.OneOf(['scan', 'policy', 'remediation'])
+    )
```

Comment:
can't we merge these 3 schemas into a single schema and then using validate using the validate_schema decorator

Reply:
that would be same as not using marshmallow.